packages/cli/src/lib/build-strategies/strategies/LocalStrategy.ts

```diff
@@ -25,14 +25,15 @@ export class LocalBuildStrategy extends BuildStrategy<void> {
 
         const command = `chmod +x ${scriptPath} && ${scriptPath} ${buildManifestConfig.polywrap_module.dir} ${this.outputDir}`;
 
-        await withSpinner(
+        await logActivity(
+          this.project.logger,
           intlMsg.lib_helpers_buildText(),
           intlMsg.lib_helpers_buildError(),
           intlMsg.lib_helpers_buildWarning(),
           async (_spinner) => {
```

Comment:
Wouldn't we want to rename this to `logger` and pass it along into line 36 as the argument for the `logger` in `runCommand`? We ARE passing along the same thing, just a stylistic preference.